lib/chef/resource/mount.rb

```diff
@@ -24,8 +24,6 @@ class Resource
     class Mount < Chef::Resource
       description "Use the mount resource to manage a mounted file system."
 
-      identity_attr :device
```

Comment:
identity needs to be set on device if this goes away

Reply:
Yep, it's there :)

Reply:
yep I'm blind